snap/snapcraft.yaml

```
@@ -0,0 +1,33 @@
+name: flamenco
+base: core22
+version: '1.0.0'
+summary: Package maintenance tool for Ubuntu Toolchains
+description: |
+  Flamenco is a CLI tool that helps toolchain developers manage many
+  different package versions and releases from a single debian
+  folder source tree.
+
+grade: stable
+confinement: strict
```

issue: This project requires to read and write from/to the host file system. Therefie it needs to be a `classic` snap.

```suggestion
confinement: classic
```

---

Do we need to read/write from/to locations outside of the user's home, which is typically where they keep their workspace?

Making it classic removes all confinements (plus triggers a manual review, IIRC), which seems a bit overkill if we're only looking to read/write from the host system (potentially home dir).

---

I am not happy about it, but I guess it's acceptable. 

For example you can't run flamenco from `/tmp/**` (which is in my opinion a reasonable thing to do):
```
$ flamenco build debian-tarball --source-directory /tmp/tmp.5mFM6DAIeF/src --destination-directory /tmp/tmp.5mFM6DAIeF/dist
[DEBUG] Source Directory: /tmp/tmp.5mFM6DAIeF/src
[DEBUG] Destination Directory: /tmp/tmp.5mFM6DAIeF/dist
[ERROR] The source directory '/tmp/tmp.5mFM6DAIeF/src' does not exist!
[FATAL] Aborting the build process, because the source directory contains errors.
```

Or it will happily accept to build in `/tmp/**`, but because of the isolation `/tmp` you will not be able to see the output:
```
$ flamenco build debian-tarball --source-directory ~/Repositories/dotnet/source-build-security/src --destination-directory /tmp/tmp.5mFM6DAIeF/dist
[DEBUG] Source Directory: /home/dviererbe/Repositories/dotnet/source-build-security/src
[DEBUG] Destination Directory: /tmp/tmp.5mFM6DAIeF/dist
[DEBUG] Discovered targets in source directory: dotnet6:noble dotnet8:jammy dotnet8:mantic dotnet6:mantic dotnet7:mantic dotnet7:noble dotnet8:noble dotnet8:oracular dotnet7:jammy dotnet6:jammy
[DEBUG] Requested build targets: None
[INFO] Building all buildable targets.
[INFO] Building target dotnet6:noble succeeded.
[INFO] Building target dotnet8:jammy succeeded.
[INFO] Building target dotnet8:mantic succeeded.
[INFO] Building target dotnet6:mantic succeeded.
[INFO] Building target dotnet7:mantic succeeded.
[INFO] Building target dotnet7:noble succeeded.
[INFO] Building target dotnet8:noble succeeded.
[INFO] Building target dotnet8:oracular succeeded.
[INFO] Building target dotnet7:jammy succeeded.
[INFO] Building target dotnet6:jammy succeeded.
[INFO] Total Builds: 10; Succeeded: 10; Failed: 0
$ ls /tmp/tmp.5mFM6DAIeF/dist
ls: cannot access '/tmp/tmp.5mFM6DAIeF/dist': No such file or directory
```